src/renderer/components/properties-panel/ComputeNodesPropertiesTab.tsx

```diff
@@ -341,7 +342,7 @@ const ComputeNodePropertiesCard = ({ node, temporalEpoch, graphName }: ComputeNo
                 <div className='node-links-wrap'>
                     <h4>Links</h4>
                     {node.getNOCLinksForNode().map((link: NOCLink) => (
-                        <LinkDetails key={link.name} link={link} graphName={graphName} showEmpty />
+                        <LinkDetails key={link.name} link={link} linkState={allLinksState[link.uid]} showEmpty />
```

Comment:
we have a method to only get a single node links, lets use that

Reply:
I'll fix that afterwards, along with the other node links.

But this is inside of a loop and requires state, so we can't call a selector directly.

Reply:
you can grab a selector from within a component, you dont have to pass it in

Reply:
while we could fix this in the next round this is an easy one to fix and an easy one to miss later